services/src/pro/users/hashmap_userdb.rs

```diff
@@ -56,6 +58,14 @@ impl UserDb for HashMapUserDb {
 
         self.users.write().await.insert(id.to_string(), user);
 
+        let mut quota_available = self.quota_available.write().await;
+        quota_available.insert(
+            id,
+            crate::util::config::get_config_element::<crate::pro::util::config::User>()?
+                .default_available_quota
+                .unwrap_or(0),
```

Comment:
maybe already deserliaze it as 0 if unset

Reply:
would also be possible but leaving it this way is more flexible right? Because we still know whether it was set to 0 explicitly or left blank.

Reply:
But this leaves the default value to each implementation, right? It is sufficient, but I just wanted to see if it is actually meant to be.